tests/Dfc.CourseDirectory.WebV2.Tests/AddressSearch/GetAddressAddressSearchServiceTests.cs

```diff
@@ -21,8 +21,8 @@ public async Task SearchByPostcode_WithValidRequest_ReturnsParsedResults()
             new HttpRequestInterceptionBuilder()
                 .Requests()
                 .ForHttps()
-                .ForHost("api.getaddress.io")
-                .ForPath("find/XX2 00X")
+                .ForHost("testing.com")
```

Comment:
```suggestion
                .ForHost("example.com")
```
https://www.whois.com/whois/testing.com is a real registered domain
vs https://www.whois.com/whois/example.com which is reserved for test usage etc.
Same for all the others below, I won't repeat the comment.

Reply:
Why does it matter in a closed unit test?

Reply:
Two reasons I can think of:

1. Eliminate the risk of tests accidentally reaching out to real servers at some point in future (you never know what other people will do with your carefully crafted code in the future)
2. Eliminate the risk that the arbitrary mention of someone else's domain becoming problematic for non-technical reasons in the future (for example if the domain you have used becomes part of a malware system, or has some massive public relations disaster).

Given that there's no downside to sticking to the reserved domains I see no reason not to do so as a matter of habit.

Reply:
Agreed.